package.json

```diff
@@ -46,28 +45,29 @@
     "babel-register": "^6.26.0",
     "chai": "^4.2.0",
     "chai-as-promised": "^7.1.1",
-    "chai-enzyme": "^0.8.0",
+    "chai-enzyme": "^1.0.0-beta.1",
     "chalk": "^3.0.0",
     "classnames": "^2.2.6",
     "coveralls": "^3.0.9",
     "css-loader": "^3.4.2",
     "cucumber": "^6.0.5",
     "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.2",
     "eslint": "^6.8.0",
     "eslint-config-brainhub": "^1.11.1",
     "eslint-watch": "^6.0.1",
     "express": "^4.17.1",
-    "extract-text-webpack-plugin": "^3.0.2",
     "file-loader": "^5.0.2",
     "gemini": "^7.5.2",
-    "html-reporter": "^5.8.0",
+    "html-reporter": "4.13.2",
```

Comment:
why do you downgrade this?

Reply:
Downgraded html-reporter to v4 as later versions do not support the gemini package (used by your `test-regression` script).